source/includes/macos/steps-configure-minio-kes-hashicorp.rst

```diff
@@ -0,0 +1,135 @@
+This procedure assumes a single local host machine running the MinIO and KES processes.
+As part of this procedure, you will:
+
+- Deploy a |KES| server configured to use `Hashicorp Vault <https://www.vaultproject.io/>`__ as the root |KMS|.
+
+- Create a new |EK| on Vault for use with |SSE|.
+
+- Deploy a MinIO server configured to use the |KES| container for supporting |SSE|.
+
+- Configure automatic bucket-default :ref:`SSE-KMS <minio-encryption-sse-kms>`.
+
+For production environments, MinIO recommends using Linux hosts. 
+See the MinIO on Linux documentation for configuring MinIO with KES and Hashicorp Vault.
+
+For production orchestrated environments, use the MinIO Kubernetes Operator to deploy a tenant with |SSE| enabled and configured for use with Hashicorp Vault.
+
+Prerequisites
+-------------
+
+.. _minio-sse-vault-prereq-vault:
+
+Hashicorp Vault
+~~~~~~~~~~~~~~~
```

Comment:
Add verbs to subheadings, as mentioned for the other platform pages.

Reply:
Done throughout.

Also removing the prereqs that are actually just behaviors.